packages/jsii-pacmak/test/expected.jsii-calc-lib/python/src/scope/jsii_calc_lib/_jsii/__init__.py

```diff
@@ -2,13 +2,14 @@
 import builtins
 import datetime
 import enum
+import publication
 import typing
 
 import jsii
 import jsii.compat
-import publication
 
 import scope.jsii_calc_base
+
```

Comment:
Is this change actually needed or can we revert? (try to reduce the blast radius of your change)